src/main/java/com/sinthoras/visualprospecting/gui/xaeromap/rendersteps/InteractableRenderStep.java

```diff
@@ -0,0 +1,18 @@
+package com.sinthoras.visualprospecting.gui.xaeromap.rendersteps;
+
+import com.sinthoras.visualprospecting.gui.model.locations.IWaypointAndLocationProvider;
+import net.minecraft.client.gui.GuiScreen;
+
+public interface InteractableRenderStep extends RenderStep {
+
+	boolean isMouseOver(double mouseX, double mouseY, double cameraX, double cameraZ, double scale);
+
+	void drawTooltip(GuiScreen gui, double mouseX, double mouseY, double cameraX, double cameraZ, double scale, int scaleAdj);
```

Comment:
Just curous: what is scaleAdj? Check the usage and it wasn't clear right away

Reply:
scaleAdj is the current screen scaling value, which there is an opengl scaling for elsewhere in the map's huge render function. It's required for the tooltip to render in the correct spot for some reason.